### DISCUSSION TITLE
Maven package with secrets

### DISCUSSION THREAD

**Discussion by lx78WyY0J5** [community/community#31574] [Actions]
NB thx to use older branch to see code while I gaved up this way

Hi, i'm trying to pass secrets to my Jar file builded with maven 
[Sorry for the code mess, it's an old project](https://github.com/lx78WyY0J5/project-plugin) reuped'

I've tried to find how to achieve it, but i' can't give the secret to the final jar

I can just pass a "secret" by using the tag `-Dname="value"` with the CMD but not the maven actions

#
[Action workflow](https://github.com/lx78WyY0J5/project-plugin/blob/main/.github/workflows/maven.yml)
Build the jar, get the secret, but IDK if it pass it to the final mapped Jar (Can't get it with code (**Null error**)

#
[Getting the value from code](https://github.com/lx78WyY0J5/project-plugin/blob/main/src/main/java/com/github/lx78WyY0J5/main/main.java#L53-L89)
Work well, only one of the two work, i've let the wrong one for testing purposes

#
[POM.xml](https://github.com/lx78WyY0J5/project-plugin/blob/main/pom.xml)
seems OK

#
Thanks a lot for help, and sorry for my bad English

**Comment by Somemyii** [comment]
Deploiement master

**Reply by lx78WyY0J5** [reply]
What do you mean by deploiement master ? It means nothing I'm already in a master deploiement script 🤔   
Thx for reply
